packages/indexer/src/services/intentGateway.service.ts

```diff
@@ -172,29 +185,46 @@ export class IntentGatewayService {
 
 			// Award points for order filling - using USD value directly
 			if (status === OrderStatus.FILLED && filler) {
+				// Volume
+				let outputPaymentInfo: PaymentInfo[] = orderPlaced.outputTokens.map((token, index) => {
+					return {
+						token: token as Hex,
+						amount: orderPlaced.outputAmounts[index],
+						beneficiary: orderPlaced.outputBeneficiaries[index] as Hex,
+					}
+				})
+				let outputUSD = await this.getOutputValuesUSD(outputPaymentInfo)
+				await VolumeService.updateVolume("IntentGateway.FILLER", outputUSD.total, timestamp)
```

Comment:
Let's index volume for the actual filler also something like:
`await VolumeService.updateVolume("IntentGateway.FILLER.{filler_address}", outputUSD.total, timestamp)`

Reply:
This good? https://github.com/polytope-labs/hyperbridge-sdk/pull/108/commits/1f8e2b55e4455ab82101e70079a3c7746085dccf